### PR TITLE
Added coloring to the panels in the failures

### PR DESCRIPTION
**Pull request by psav**
Added coloring to the panels in the failures

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit dfc8f28f442b9ddc3190077ecd823eb9c26ae205:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: data/templates/test_report.html [hunk 1]
```diff
@@ -88,7 +88,13 @@
...
 
 {% for test in tests %}
-    <div class="panel panel-info">
+    <div class="panel panel-{% if test.outcomes['overall'] == 'passed' %}success
+                      {% elif test.outcomes['overall'] == 'failed' %}danger
+                      {% elif test.outcomes['overall'] == 'skipped' %}info
+                      {% elif test.outcomes['overall'] == 'error' %}danger
+                      {% elif test.outcomes['overall'] == 'xpassed' %}danger
+                      {% elif test.outcomes['overall'] == 'xfailed' %}success
+                      {% endif %}">
```
Comment (mfalesni) [inline data/templates/test_report.html]: that would deserve some filter :)

Comment (seandst) [inline data/templates/test_report.html]: Yeah, the template isn't the place for this.

Comment (psav) [inline data/templates/test_report.html]: OK, I will build a color var

**Comment by dajoRH** [comment]
Lint report for commit 65111ed0b2ef93b6517e3c3b392ad93f58b906fb:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4f171b4e86b934e9e57fb2c9d9aa133c2a5456ee:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**